mysite/psa/views.py

```diff
@@ -164,3 +166,39 @@ def set_pass(request):
         return context(changed=True, person=user)
     else:
         return context(exception='Something goes wrong...', person=user)
+
+
+def forgot_pass(request):
+    email = request.POST['email']
+    users = User.objects.filter(email=email)
+    #import pdb;pdb.set_trace()
+    if len(users) > 0:
```

Comment:
This can be written as:

    user = User.objects.filter(email=email).first()
    if user:
        token = TokenForgotPassword(user=user)
        token.save()
        url = request.build_absolute_uri(token.get_absolute_url())
        send_forgot_password(user, url)
        return JsonResponse({"success": True})
    else:
        return JsonResponse({
            "success": False,
            "error": 'User with email {0} not found'.format(email)
        })